### DISCUSSION TITLE
socket.io SSL at Arduino MKR1010

### DISCUSSION THREAD

**Discussion by RQnet** [khoih-prog/WebSockets_Generic#20] [Q&A]
I like your librarie. It works verry well with out ssl. I can connect with "socketIO.begin(..)." but with "socketIO.beginSSL(...)" it doesn´t work with my MKR1010. I got just this message:

[WS] WebSockets_Generic v2.12.0
[WS] [wsIOc] found EIO=4 disable EIO ping on client
[WS] [WS-Client] Connect wss...
[WN] r:Check&WLost
[WN] r:WLost.ReconW
.....
....
disconnecting from server.

in my case i can´t use it. SSL is required. 

Is it planned to implement the support for MKR1010?

**Comment by khoih-prog** [comment]
I have no plan to add SSL support to this **outdated** MKR1010 board. I also don't have any and no plan to buy any MKR1010 to test.

Try to switch to ESP32/ESP8266, WT32_ETH01, or Nano_RP2040_Connect.

**Reply by RQnet** [reply]
SSL works very well on MKR1010.

it was a problem of my own implementation.
